marketplaceItems/drawApp/fingerDraw_app.js

```diff
@@ -0,0 +1,596 @@
+                    triggerReleased();
+                }
+            }
+        });
+        controllerMapping.from(Controller.Standard.RightGrip).to(function (value) {
+            if (dominantHand === "right") {
+                if (value >= MINIMUM_TRIGGER_PRESS_VALUE && !activeGripPress) {
+                    activeGripPress = true;
+                    gripPressed();
+                } else if (value <= MINIMUM_TRIGGER_PRESS_VALUE && activeGripPress) {
+                    gripReleased();
+                }
+            }
+        });
+        controllerMapping.from(Controller.Standard.LT).to(function (value) {
+            if (dominantHand === "left") {
+                if (value >= MINIMUM_TRIGGER_PRESS_VALUE && !activeTriggerPress ) {
+                    activeTriggerPress = true;
+                    triggerPressed();
+                } else if (value <= MINIMUM_TRIGGER_PRESS_VALUE && activeTriggerPress) {
+                    triggerReleased();
+                }
+            }
+        });
+        controllerMapping.from(Controller.Standard.LeftGrip).to(function (value) {
+            if (dominantHand === "left") {
+                if (value >= MINIMUM_TRIGGER_PRESS_VALUE && !activeGripPress) {
+                    activeGripPress = true;
+                    gripPressed();
+                } else if (value <= MINIMUM_TRIGGER_PRESS_VALUE && activeGripPress) {
+                    gripReleased();
+                }
+            }
+        });
+    }
+
+    /* ON TRIGGER PRESS DRAW: Store the initial point and begin checking distance hand has moved on an interval. If hand 
+    has moved more than minimum distance, draw a polyline entity with a lifetime of 1 minute and continue checking 
+    hand distance. Every time hand moves more than the minumum, update the polyline with another node. */
+    function triggerPressed() {
+        if (tablet.tabletShown || activeGripPress) {
+            return;
+        }
+        lineStartPosition = MyAvatar.getJointPosition(parentJointIndex);
+        previousLinePoint = MyAvatar.getJointPosition(parentJointIndex);
+        distanceCheckInterval = Script.setInterval(function() {
+            var currentLinePoint = MyAvatar.getJointPosition(parentJointIndex);
+            if (Vec3.distance(previousLinePoint, currentLinePoint) > MINIMUM_MOVEMENT_TO_DRAW_M) {
+                var displacementFromStart = Vec3.subtract(currentLinePoint, lineStartPosition);
+                linePoints.push(displacementFromStart);
+                if (!polyLine) {
+                    playSound(DRAW_SOUND, DRAW_SOUND_VOLUME, currentLinePoint, false, true);
+                    if (paintSphere) {
+                        paintSphereDimensions = Entities.getEntityProperties(paintSphere, 'dimensions').dimensions;
+                    }
+                    lineStrokeWidths = [paintSphereDimensions.x, paintSphereDimensions.x];
+                    polyLine = Entities.addEntity({
+                        type: "PolyLine",
```

Comment:
these properties are repeated four times in the code - could you please make these properties a `var CONST` above where they're first used?

Reply:
the paint sphere dimensions are checked every time, and change with the avatar size, so it would not be constant

Reply:
many of the properties are though, right? like type, name, color, collisionless, etc

Reply:
oh, sorry, I thought you meant the paintSphereDimensions.x var...will fix!